packages/vats/decentral-devnet-config.json

```diff
@@ -83,6 +125,18 @@
     "board": {
       "sourceSpec": "@agoric/vats/src/vat-board.js"
     },
+    "ibc": {
+      "sourceSpec": "@agoric/vats/src/vat-ibc.js"
+    },
+    "network": {
+      "sourceSpec": "@agoric/vats/src/vat-network.js"
+    },
```

Comment:
@ivanlei the [list of vats/contracts](https://github.com/Agoric/agoric-sdk/issues/5014#issuecomment-1309365804) that I put together Nov 9 for planning purposes didn't include these two (ibc, network). I suppose they should be added to the list of things to make upgradeable (#6553). Ah... we already have #5696 for the network vat. I don't know whether we included it in any plans.